index.html

```diff
@@ -18,7 +18,7 @@
   <body>
     <header>
       <!-- Nav bar -->
-      <nav class="navbar navbar-expand-lg navbar-light bg-light custom-nav">
+      <nav class="navbar navbar-expand-lg navbar-light bg-danger custom-nav">
```

Comment:
Please could you set this to be the colour we agreed in our colour palette #9A031E